app/api/speaker_image_sizes.py

```diff
@@ -0,0 +1,19 @@
+from flask_rest_jsonapi import ResourceDetail
+
+from app.api.bootstrap import api
+from app.api.schema.image_sizes import SpeakerImageSizeSchema
+from app.models import db
+from app.models.image_size import ImageSizes
+
+
+class SpeakerImageSizeDetail(ResourceDetail):
+    """
+    Speaker Image_size detail by id
+    """
+    def before_get(self, args, kwargs):
+        kwargs['id'] = 2
+    decorators = (api.has_permission('is_admin', methods="PATCH", id="2"),)
```

Comment:
why id = 2 ?

Reply:
id=1 is used for EventimageSize obj and id=2 is used for SpeakerImageSize obj as the two objects created in populate_db.py.

Reply:
Couldn't we use ENUMS instead o fnumbers?

Reply:
ENUMS in which part @Kreijstal  ??